packages/cli/test/integration.js

```diff
@@ -3909,3 +3926,159 @@ test('[vc build] should build project with `@vercel/static-build`', async t => {
+  await waitForPrompt(vc, chunk => chunk.includes('Linked to'));
+  const deployment = await vc;
+  t.is(deployment.exitCode, 0, formatOutput(deployment));
+  // assert the command were executed
+  let page = await fetch(deployment.stdout);
+  let text = await page.text();
+  t.is(text, '1\n');
+});
+
+test('vercel.json configuration overrides in an existing project do not prompt user and correctly apply overrides', async t => {
+  // create project directory and get path to vercel.json
+  const directory = fixture('vercel-json-configuration-overrides');
+  const vercelJsonPath = path.join(directory, 'vercel.json');
+
+  async function deploy(autoConfirm = false) {
+    const deployment = await execa(
+      binaryPath,
+      [directory, ...defaultArgs, '--public'].concat(
+        autoConfirm ? ['--confirm'] : []
+      ),
+      { reject: false }
+    );
+    t.is(
+      deployment.exitCode,
+      0,
+      formatOutput({
+        stderr: deployment.stderr,
+        stdout: deployment.stdout,
+      })
+    );
+    return deployment;
+  }
+
+  // Step 1. Create a simple static deployment with no configuration.
+  // Deployment should succeed and page should display "0"
+
+  await mkdir(path.join(directory, 'public'));
+  await writeFile(path.join(directory, 'public/index.txt'), '0\n');
+
+  // auto-confirm this deployment
+  let deployment = await deploy(true);
+
+  let page = await fetch(deployment.stdout);
+  let text = await page.text();
+  t.is(text, '0\n');
+
+  // Step 2. Now that the project exists, override the buildCommand and outputDirectory.
+  // The CLI should not prompt the user about the overrides.
+
+  const BUILD_COMMAND = 'mkdir output && echo "1" >> output/index.txt';
+  const OUTPUT_DIRECTORY = 'output';
+
+  await writeFile(
+    vercelJsonPath,
+    JSON.stringify({
+      buildCommand: BUILD_COMMAND,
+      outputDirectory: OUTPUT_DIRECTORY,
+    })
+  );
+
+  deployment = await deploy();
+  page = await fetch(deployment.stdout);
+  text = await page.text();
+  t.is(text, '1\n');
+
+  // This isn't working for some reason.
+  // // Step 3. Do a more complex deployment using a framework this time
+
+  // // Test Next.js Framework deployment
+  // await mkdir(`${directory}/pages`);
+  // await writeFile(
+  //   `${directory}/pages/index.js`,
+  //   `export default () => '2\n'`
+  // );
+  // await writeFile(
+  //   vercelJsonPath,
+  //   JSON.stringify({
+  //     framework: 'nextjs',
+  //   })
+  // );
+  // await writeFile(
+  //   `${directory}/package.json`,
+  //   JSON.stringify({
+  //     scripts: {
+  //       dev: 'next',
+  //       start: 'next start',
+  //       build: 'next build',
+  //     },
+  //     dependencies: {
+  //       next: 'latest',
+  //       react: 'latest',
+  //       'react-dom': 'latest',
+  //     },
+  //   })
+  // );
+
+  // deployment = await deploy();
+  // page = await fetch(deployment.stdout);
+  // text = await page.text();
+  // t.is(text, '2\n');
```

Comment:
What's the failure when this code is run?

Reply:
`Error! Command "yarn run build" exited with 1`

Reply:
fixed!